frontend/src/features/idir/search/components/IDIRPermitSearchResults.tsx

```diff
@@ -177,6 +177,7 @@ export const IDIRPermitSearchResults = memo(
                 permitNumber={row.original.permitNumber}
                 permitId={row.original.permitId}
                 userAuthGroup={idirUserDetails?.userAuthGroup}
+                companyId={row.original.companyId?.toString()}
```

Comment:
Hmm... Is this safe? If companyId is undefined, then what would happen if it pass the string "undefined"?

Reply:
Should we make it something like ```getDefaultRequiredVal("0", row.original.companyId?.toString())``` instead?

Reply:
With optional chaining, you can safely call a function on an undefined property. 

`x?.toString()` basically transpiles to something like `x !== null && x !== undefined && x.toString()`.

The API response includes companyId and the API lists it as a required 
field. companyId is central to everything we do w.r.t permits currently and hence guaranteed to exist.

Adding 0 isn't something I think is necessary. If the value is undefined, the API url simply translates to `companies//permits/{permitId}` and the API throws an 404.

Reply:
Ok no worries then